Pipfile

```diff
@@ -8,7 +8,7 @@ python_version = "3.10"
 
 [packages]
 Flask = ">=2.3.2,<2.4"
-PyYAML = ">=5,<7"
+PyYAML = "*"
```

Comment:
I mean just remove it completely, since we don't actually use it directly in our code.